gp-limit-submissions/gpls-disable-rule-for-user-role.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * Gravity Perks // GP Limit Submissions // Disable Rule for a Specific User Role
```

Comment:
Nitpicky but since this will essentially dislabe all limit feeds for a given form, it might be a clearer to change the name to "Disable Limits for a Specific User Role". If you agree, let's update the file name as well.